python/composio/tools/toolset.py

```diff
@@ -628,6 +650,22 @@ def _process_schema_properties(self, action: Action, properties: t.Dict) -> t.Di
             type_="schema",
         )
 
+    def _merge_processors(self, processors: ProcessorsType) -> None:
```

Comment:
Consider adding a type hint for the `processors` parameter in the `_merge_processors` method to improve code readability and maintainability.